source/features/hidden-review-comments-indicator.css

```diff
@@ -8,7 +8,7 @@
 
 .rgh-comments-indicator:is(::before, ::after) {
 	content: '' !important; /* Needs to override .blob-num’s */
-	position: absolute;
+	position: absolute !important; /* Needs to override .blob-num’s */
```

Comment:
Is this selector doing anything? I don't see the line added by `border-top` in the attached screenshot

Reply:
The class is working, but I'm not so sure about the selector, let me do some testing :)